libs/ledger-live-common/src/families/evm/api/etherscan.ts

```diff
@@ -69,13 +72,53 @@ export const getLatestTransactions = makeLRUCache<
     });
 
     return ops
-      .map((tx) => etherscanOperationToOperation(accountId, address, tx))
+      .map((tx) => etherscanOperationToOperation(accountId, tx))
       .filter(Boolean) as Operation[];
   },
   (currency, address, accountId) => accountId,
   { maxAge: 6 * 1000 }
 );
 
+/**
+ * Get all the last ERC20 transactions
+ */
+export const getLastTokenOperations = makeLRUCache<
+  [
+    currency: CryptoCurrency,
+    address: string,
+    accountId: string,
+    fromBlock: number
+  ],
+  { tokenCurrency: TokenCurrency; operation: Operation }[]
```

Comment:
Just a personal opinion but could be nice to give name to types to make it easier what they represent and also makes the code a bit easier to read

```ts
type MyFooType = [
    currency: CryptoCurrency,
    address: string,
    accountId: string,
    fromBlock: number
 ];

type CachedOperation = { tokenCurrency: TokenCurrency; operation: Operation };

export const getLastTokenOperations = makeLRUCache<MyFooType, CachedOperation[]>
```